src/formatDate.js

```diff
@@ -8,7 +8,49 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // '2020-02-18', ['YYYY', 'MM', 'DD', '-'], ['DD', 'MM', 'YY', '.']
+
+  const separatorFrom = fromFormat[fromFormat.length - 1]; // '-',
+  const separatorToFormat = toFormat[toFormat.length - 1]; // '.'
+
+  const formatedYear = [];
```

Comment:
The variable `formatedYear` is misleading since it doesn't only store the year but will eventually store all parts of the formatted date. A name that reflects its purpose more accurately, such as `formattedDateParts`, would be more appropriate.